src/libsyntax/ast.rs

```diff
@@ -168,10 +169,10 @@ pub const ILLEGAL_CTXT : SyntaxContext = 1;
 pub struct Name(pub u32);
 
 impl Name {
+    #[allow(deprecated)]
```

Comment:
This is using a deprecated function that's slated for removal, could it avoid that?